app/controllers/bids_controller.rb

```diff
@@ -0,0 +1,13 @@
+class BidsController < ApplicationController
+  include Secured
+
+  def create
+    user_id = current_user.id
+    auction_id = params[:id]
+    body = JSON.parse(request.body.read, symbolize_names: true)
```

Comment:
Add `params.require` (see docs) to send a 400 if params are missing

Reply:
@ijpiantanida Is this still helpful if the route is changed to `/auctions/:auction_id/bid/`?